.github/workflows/tests.yml

```diff
@@ -269,3 +269,41 @@ jobs:
           override: true
       - name: Run cargo deny
         uses: EmbarkStudios/cargo-deny-action@v1
+
+  packages:
```

Comment:
I think we should move it into another config file, we probably don't want to run it on every pull request?

Reply:
Actually, I switched it back to running on PRs too — I think we want to find out early if we broke something in our packaging process, like taking a dependency on a too-new crate or something that doesn't build in our packaging environment.

Reply:
Good point, makes sense.